### ISSUE TITLE
Add tests for accessibility use cases

### ISSUE THREAD

**Issue by BenHenning** [google/blockly#8915]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Problem

There aren't yet end-to-end tests for verifying the new keyboard navigation and accessibility use cases in Core Blockly (though they have started to be implemented for navigation in the keyboard navigation plugin repository: https://github.com/google/blockly-keyboard-experimentation/tree/main/test).

### Request

WebdriverIO tests should be added that verify both keyboard navigation & readout state as more of the keyboard navigation plugin and navigation functionality is moved into Core Blockly.

**Comment by BenHenning** [comment]
Reminder to consider shadow block cases specifically as they relate to selection & focus per https://github.com/google/blockly/pull/8916#discussion_r2056747205.

**Comment by BenHenning** [comment]
From #8923: make sure to add tests for drop-down & widget divs in how they manage their ephemeral focus.

**Comment by BenHenning** [comment]
https://github.com/google/blockly/issues/8826#issuecomment-2860837080 another reference point.

**Comment by BenHenning** [comment]
Note that I plan to address this issue in two pats:
1. Identifying and shoring up existing (regression) tests for cases that were found while working on v12-related features (especially around focus).
2. Identifying top-level classes of user flows (likely more from an end-to-end perspective) to ensure thorough accessibility testing. The latter should complement existing plans and tests within the keyboard navigation plugin repo, plus consider screen reader label outputs (though some of both of these can also be tested more thoroughly with unit tests--it's likely there will be two categories of new accessibility-specific tests to be added here).

**Comment by rachel-fenichel** [comment]
A list of test cases for copy/paste behaviour: https://github.com/google/blockly-keyboard-experimentation/issues/103#issuecomment-2881792242

**Comment by gonfunko** [comment]
Test for committing moves on keyboard shortcuts: https://github.com/google/blockly-keyboard-experimentation/pull/564#pullrequestreview-2879029449

**Comment by gonfunko** [comment]
Closing now that v13 is out and we have decent test coverage for that; the one remaining open subissue can track that specific piece of work.
